### PR TITLE
Stop using deprecated Order.incr and Order.decr #1562

### LINKED ISSUE #1562: Stop using deprecated Order.incr and Order.decr enum values
TinkerPop deprecated `Order.incr` and `Order.decr` [in version 3.3.4](http://tinkerpop.apache.org/docs/3.4.0/upgrade/#_introducing_order_asc_and_order_desc) and replaced them with `Order.asc` and `Order.desc`.
We should replace all mentions of the deprecated `Order` enum values with the new ones in the docs and also everywhere in the code.

### PR DESCRIPTION
**Pull request by porunov**
Fixes #1562

Signed-off-by: Oleksandr Porunov <alexandr.porunov@gmail.com>

-----

Thank you for contributing to JanusGraph!

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [x] Is there an issue associated with this PR? Is it referenced in the commit message?
- [x] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [x] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [x] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you written and/or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](https://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE.txt file, including the main LICENSE.txt file in the root of this repository?
- [ ] If applicable, have you updated the NOTICE.txt file, including the main NOTICE.txt file found in the root of this repository?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?
- [ ] If this PR is a documentation-only change, have you added a `[skip ci]`
  tag to the first line of your commit message to avoid spending CPU cycles in
  Travis CI when no code, tests, or build configuration are modified?

### Note:
Please ensure that once the PR is submitted, you check Travis CI for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

**Comment by porunov** [comment]
Thank you for pointing this. I've changed documentation to use `asc` and `desc` as well as field names.

File: janusgraph-lucene/src/test/java/org/janusgraph/diskstorage/lucene/BerkeleyLuceneTest.java [hunk 1]
```diff
@@ -67,30 +67,30 @@ public void testPrintSchemaElements() {
         GraphOfTheGodsFactory.load(graph);
         mgmt = graph.openManagement();
 
-        String expected = "99EB9D82CE85D15A9120E31969B3F861";
+        String expected = "A18422278042949EE2B162837EC27A63";
```
Comment (FlorianHockmann) [inline janusgraph-lucene/src/test/java/org/janusgraph/diskstorage/lucene/BerkeleyLuceneTest.java]: Do you know why these hashes changed?

Comment (porunov) [reply janusgraph-lucene/src/test/java/org/janusgraph/diskstorage/lucene/BerkeleyLuceneTest.java]: Yes. It also prints some index order. Which was `incr` and `decr` and now it is `asc` and `desc`. That is why the output hash has been changed

File: docs/generating.adoc [hunk 2]
```diff
@@ -38,4 +38,4 @@ size = 100000; ids = [g.addVertex().id()]; rand = new Random()
 === Computing the In-Degree Distribution of the Graph
 
 [source, gremlin]
-g.V().local(inE().count()).groupCount().order(local).by(valueDecr)
+g.V().local(inE().count()).groupCount().order(local).by(desc)
```
[the remark this answers is not in the document]
Comment (porunov) [reply docs/generating.adoc]: You are right. I've changed it.

**Pull request merged**